src/client/Microsoft.Identity.Client/Cache/Items/CacheSerializationContract.cs

```diff
@@ -4,13 +4,27 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+#if NET6_0_OR_GREATER
+using System.Text.Json;
+using System.Text.Json.Nodes;
+using JObject = System.Text.Json.Nodes.JsonObject;
+using JToken = System.Text.Json.Nodes.JsonNode;
+#else
 using Microsoft.Identity.Json;
 using Microsoft.Identity.Json.Linq;
+#endif
 
 namespace Microsoft.Identity.Client.Cache.Items
 {
     internal class CacheSerializationContract
     {
+#if NET6_0_OR_GREATER
+        private static readonly JsonSerializerOptions NeverIgnoreJsonOptions = new()
```

Comment:
Interesting point from the Frugal Tips discussions is that these options should be static/reused, since they have some sort of resource intensive initialization code behind the scenes.